tests/unit/profiles/profilesService.test.ts

```diff
@@ -49,6 +49,126 @@ describe('Profiles Service', () => {
     expect(res.data.length).toBeGreaterThan(0);
   });
 
+  it('should not convert the custom fields to camel case', async () => {
+    const profile = {
+      id: profileId,
+      custom_fields: {
+        hello_world: 'log hello world',
+        property_1: 'jeej!',
+      },
+    };
+
+    const rql = rqlBuilder().build();
+    nock(`${host}${PROFILES_BASE}`)
+      .get('/')
+      .reply(200, createPagedResponse(profile));
+
+    const res = await sdk.profiles.find({ rql });
+
+    expect(res.data[0].customFields).toStrictEqual(profile.custom_fields);
+  });
+
+  it('should convert the custom fields to camel case if the skipOption is turned off', async () => {
+    const profile = {
+      id: profileId,
+      custom_fields: {
+        hello_world: 'log hello world',
+        property_1: 'jeej!',
+      },
+    };
+
+    const rql = rqlBuilder().build();
+    nock(`${host}${PROFILES_BASE}`)
+      .get('/')
+      .reply(200, createPagedResponse(profile));
+
+    const res = await sdk.profiles.find({
+      rql,
+      skipCaseNormalizationForCustomProperties: false,
```

Comment:
Are we sure want to keep calling this with the `skip` making it a double negative.
We could call it `normalizationCasingForCustomProperties` and default it to false instead of true, that would be easier to reason with.